### ISSUE TITLE
Sigar on Arch prevents Build

### ISSUE THREAD

**Issue by Sparkier** [inviwo/inviwo#238]
**Issue**

Inviwo fails to build on Arch because of sigar.
Errors are the following:
```bash
inviwo/ext/sigar/src/os/linux/linux_sigar.c:1149:22: 
error: called object ‘major’ is not a function or function pointer 
#define ST_MAJOR(sb) major((sb).st_rdev)

inviwo/ext/sigar/src/os/linux/linux_sigar.c:1150:22: 
error: called object ‘minor’ is not a function or function pointer 
#define ST_MINOR(sb) minor((sb).st_rdev)
```
along with other occurences in other lines of the same file.

**Steps to reproduce**

```
git clone https://github.com/inviwo/inviwo.git
git submodule update --init --recursive
mkdir build && cd build
cmake ..
make inviwo
```

**Environment information**
* Operating System: Arch Linux
* Compiler/IDE: gcc
* QT Version: 5.11.1
* Python version: 3.7.0

**Comment by petersteneteg** [comment]
:( there should be a cmake option to disable sigar I think... 
unfortunately sigar is very old and not updated anymore...

**Comment by ResearchDaniel** [comment]
Disable IVW_SIGAR
Could we detect the linux version and disable it by default?

**Comment by petersteneteg** [comment]
Maybe we should just get rid of it...

**Comment by petersteneteg** [comment]
Think this should be fixed by #601
